packages/xo-server-test/package.json

```diff
@@ -36,13 +36,14 @@
     "golike-defer": "^0.4.1",
     "jest": "^24.8.0",
     "lodash": "^4.17.11",
+    "promise-toolbox": "^0.13.0",
     "xo-collection": "^0.4.1",
     "xo-common": "^0.2.0",
     "xo-lib": "^0.9.0"
   },
   "scripts": {
-    "dev-test": "jest --bail --watch",
-    "test": "jest"
+    "dev-test": "jest --bail --watch --runInBand",
+    "test": "jest --runInBand"
```

Comment:
I think we can improve this solution by running only the necessary tests sequentially. The others will always run in parallel.

We can talk about it later if for you, it's interesting to do it.

Reply:
It's a good idea. Have you any solution to let jest select which test should run in parallel?

Reply:
```
// package.json
...
"scripts": {
    "dev-test": "jest --bail --watch --runInBand",
    "test": "jest && jest --runInBand backupNg && jest --runInBand job"
}
  "jest": {
    "modulePathIgnorePatterns": [
      "<rootDir>/src/old-tests",
      // add files run sequentially
      "<rootDir>/src/backupNg",
      "<rootDir>/src/job",
    ],
    "testEnvironment": "node",
    "testRegex": "\\.spec\\.js$"
  }
...
```

I did not test it but I think it can work.

Reply:
I just read your comment below. and if I understand correctly it is better to not modify the predefined commands of test. So the previous solution is not really good.